File: Modulo-03/Agencia-Viagem-bootstrap-C-Sharp-Core-Entity/BrasTravel/BrasTravel/Models/Viagem.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BrasTravel.Models
{
    [Table("Viagem")]
    public class Viagem
    {

        [Key]
        public int IdViagem { get; set; }

        [Required(ErrorMessage = "Informe a origem de sua viagem.")]
        public string Origem { get; set; }

        public string Destino { get; set; }

        public string Valor { get; set; }

        [Required(ErrorMessage = "Informe sua Data de Ida.")]
       
        public DateTime DataIda { get; set; }

        
        public DateTime DataVolta { get; set; }

        public virtual Pessoa Pessoa { get; set; }



    }


}

File: Modulo-05/brastravel Back-end/src/main/java/br/com/brastravel/model/Viagem.java
package br.com.brastravel.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Date;
import java.time.Instant;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "viagem")
public class Viagem {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Integer id;
	
	private String origem;
	private String destino;
	private String valor;
	private Date data_ida;
	private Date data_volta;

}
